test/integration/checkout_int_test.go

```diff
@@ -387,6 +387,26 @@ func (suite *CheckoutIntegrationTestSuite) TestNoLanguage() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *CheckoutIntegrationTestSuite) TestChangeSummary() {
+	if runtime.GOOS == "windows" {
+		suite.T().Skip("cp.SendCtrlC() does not work on Windows")
+	}
+	suite.OnlyRunForTags(tagsuite.Checkout)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/small-python"),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
```

Comment:
Suggest instead using async runtimes, so we don't actually need to source this runtime. We only need to resolve dependencies which async runtimes should be able to do.